app/views/gardens/index.html.haml

```diff
@@ -22,43 +21,38 @@
 
   %table.table.table-striped
     %tr
+      %th 
       - unless @owner
         %th Owner
       %th Garden name
-      %th Description
-      %th Location
       %th Area
-      %th Active?
+      %th Location
       %th Plantings
-      %th
 
     - @gardens.each do |garden|
       %tr
+        
+        %td.status
```

Comment:
Hi @tygriffin yes the reason is in the css. 
I need to target .label-success just in this page and to do that i'm using .status .label-success.
I didn't use .active because it was changing the color of the column